tests/spread/core22/components/simple/snap/snapcraft.yaml

```diff
@@ -31,4 +27,4 @@ parts:
       - libcurl4-openssl-dev
     stage-packages: [libcurl4]
     organize:
-      (default)/usr/share/man: (component/man-pages)
+      (default)/usr/share: (component/share)/usr/share
```

Comment:
Changing the base from `devel` to `core22` changed the contents of the `usr/share/man` directory because the stage and build package deps changed.

I just changed this from `usr/share/man` to `usr/share` to make this spread test less sensitive to the base.